reana_workflow_engine_yadage/externalbackend.py

```diff
@@ -109,7 +109,8 @@ def submit(self, spec, parameters, context):
 
         log.info('submitting!')
 
-        job_id = submit.submit('atlas', image, wrapped_cmd)
+        job_id = submit.submit(os.getenv('EXPERIMENT', 'default'),
```

Comment:
Could be dangerous to have only `EXPERIMENT` without leading prefix...

E.g. 

- `REANA_<COMPONENT>_<PROPERTY>` for module-local cfg variables
- `REANA_<PROPERTY>` for reana-global cfg variables